pkg/install/service.go

```diff
@@ -65,7 +65,10 @@ func EnsureService(args []string) error {
 	svcConfig.Dependencies = deps
 	svcConfig.Arguments = args
 
-	logrus.Info("Installing k0s service")
+	fmt.Println("Installing k0s service")
+	fmt.Println("Service information:", s.Platform())
```

Comment:
I intentionally put fmt to print out these values since I was not able to remove time stamps from logrus.Infof